src/main/java/life/qbic/io/commandline/AuthenticationOptions.java

```diff
@@ -76,4 +114,42 @@ public String toString() {
         .toString();
     //ATTENTION: do not expose the password here!
   }
+
+  public static class ReadProperties {
+
+    public static TreeMap<String, String> getProperties(String infile) {
+      final int lhs = 0;
+      final int rhs = 1;
+
+      TreeMap<String, String> map = new TreeMap<String, String>();
+      BufferedReader  bfr = null;
+      try {
+        bfr = new BufferedReader(new FileReader(new File(infile)));
+      } catch (FileNotFoundException e) {
+        throw new RuntimeException(e);
+      }
+
+      String line;
+      while (true) {
+        try {
+          if (!((line = bfr.readLine()) != null))
+            break;
+        } catch (IOException e) {
+          throw new RuntimeException(e);
+        }
+        if (!line.startsWith("#") && !line.isEmpty()) {
+          String[] pair = line.trim().split("=");
+          map.put(pair[lhs].trim(), pair[rhs].trim());
```

Comment:
NIT: 
I'd argue you don't need to initialize the ints as final variables. 
If you want it though i'd go for more descriptive names such as 
`propertyKey and propertyValue`
```suggestion
          map.put(pair[0].trim(), pair[1].trim());
```